NomadicNibbler/foodtruck-fe
# Fix/pwa offline ability

## What does this PR do?

- This adds more files the cache list in sw.js
   - the manifest file was added as that was throwing errors in the deployed application when offline see [this link](https://stackoverflow.com/questions/45463181/should-i-cache-manifest-json-in-service-worker#:~:text=Yes%20you%20should%20cache%20your,a%20200%20response%20when%20offline.)
   - the favicon images were added see: [this link](https://stackoverflow.com/questions/66971402/does-a-service-worker-need-to-cache-manifest-icons-that-are-not-used-by-the-core)
   - additional files: main.chunk.js, vendors~main.chunk.js, bundle.js

## How should it be tested?

-  pull down the branch and make sure your cache is clear and the service worker is unregistered
- do a hard refresh
- check the cache to see all files loaded
- go to application tab and click on service worker, then click offline, then refresh
- the page should be loading with static assets
- Check the network tab to see if there are any errors

## What do the future iterations hold?
-  We may need to add more files/remove files, I am not sure how neccessary the main.chunk, vendors~main.chunk, and bundle files are. I am researching into those files

## Linked Issues
- #47
